pkgs/by-name/de/deno/patches/tests-no-chown.patch

```diff
@@ -0,0 +1,77 @@
+Disable chown, uid and gid tests on Linux. These fail in the Nix sandbox.
```

Comment:
Is it worth skipping these tests completely?
I cant remember the status of darwin sandboxing but it might just be worth keeping the platforms standard and skipping these

Reply:
I'm also not fully familiar with the Darwin sandbox, but these tests do pass on Darwin for some reason. We could also completely drop them, I don't have a strong opinion.

Reply:
Are they are passing with the sandbox on on darwin? nix recently got improved support for larger (in terms of amount of store paths involved) sandboxed builds on darwin.

Reply:
Yes, these tests did pass on Darwin with `sandbox = relaxed`.